README.md

```diff
@@ -220,6 +220,78 @@ SECRET_KEY_SEED=z1Ajrg3wpDYiKS1EKAZKDd8qwR5mG3wKvh6gYi5YEz9T89P ./did key create
 }
 ```
 
+### Anchored Resources
+
+Parameters:
+- Input: Either pass in a `--file` parameter, pipe something in via `stdin`,
+  or provide a `--url` parameter (which will be fetched).
+- `--file, -f` - Optional input file containing the resource to be anchored/hashed.
+- `--canonicalize` - Optional `canonicalizationAlgorithm` value. The resource
+  to be anchored will first be canonicalized with this algorithm. For JSON or
+  JSON-LD type resources, the default algorithm is `jcs`.
+- `url` - Optional URL parameter for the anchored resource. If no input `--file`
+  or `stin` value is specified, the CLI will attempt to retrieve the resource
+  at the `url`.
+- Id: If a `url` param is provided, it will be used as the `anchoredResource.id`.
+  Otherwise, the CLI will use the `id` property of the input resource.
+  If neither exists (or if the resource is not a JSON object), no id will be
+  used.
+
+#### Creating an `anchoredResource`
+
+To create an anchored resource (which includes a content hash and
+a canonicalization algorithm): 
+
+```
+./did anchor create -f example-vc.json --canonicalize jcs
+{
+  "id": "<id of the object in example-vc.json>",
+  "canonicalizationAlgorithm": "jcs",
+  "contentHash": "z1Aaj5A4UCsdMpXwdYAReXa4bxWYiKJtdAvB1zMzCHtCbtD"
+}
+```
+
+From URL:
+
+```
+./did anchor create --url 'https://w3id.org/security/v1'
+{
+  "id": "https://w3id.org/security/v1",
+  "contentHash": "zQmdfTbBqBPQ7VNxZEYEj14VmRuZBkqFbiwReogJgS1zR1n",
+  "canonicalizationAlgorithm": "jcs"
```

Comment:
-1 to this approach because it means the `canonicalizationAlgorithm` is a property of the resource, which doesn't make sense. It's a property of the hash. That means that the hash should either become an object with a `digestValue` and a `canonicalizationAlgorithm` -- or we should find a way to get it into the multicodec encoding header.

Reply:
@dlongley I see. :/ (One _could_ argue that it's a property of the `anchoredResource`, of the anchor itself.)

Anyways, then we best go with splitting out `contentHash` into its own object.
I don't see any good way of fundamentally changing multihash to also include the canon algorithm.
I'll make the edits.

Reply:
@dlongley I updated the code and the README examples, so now it's:

```js
{
  "anchoredResource": {
    "id": "https://w3id.org/security/v1",
    "contentHash": {
      "digestValue": "zQmdfTbBqBPQ7VNxZEYEj14VmRuZBkqFbiwReogJgS1zR1n",
      "canonicalizationAlgorithm": "jcs"
    }
  }
}
```